sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -103,6 +104,14 @@ export class Receiver {
    */
   receive(onMessage: OnMessage, onError: OnError, cancellationToken?: Aborter): ReceiveHandler {
     this._throwIfReceiverOrConnectionClosed();
+    this._throwIfAlreadyReceiving();
+    let lastBatchingReceiverSequenceNum: number | undefined;
```

Comment:
What about the case when
- streaming receiver called
- streaming receiver stopped
- streaming receiver called again.

The event position in the receiver options will not be updated, right?

Reply:
Ahh, Forgot to test this case 😑